readme.md

```diff
@@ -50,6 +57,10 @@ Add the following to your POM file:
          <useTestNG>false</useTestNG>
          <!-- The naming scheme to use for the generated test classes.  One of 'simple' or 'feature-title' --> 
          <namingScheme>simple</namingScheme>
+         <!-- Generate ReRun runners instead of JUnit and TestNG. --> 
+         <useReRun>true</useReRun>
```

Comment:
Ok, there are now two properties defining which template to use: useTestNG and useReRun.

This adds complication - should rerun + testNG be supported together?